src/gui/accountsettings.cpp

```diff
@@ -985,6 +985,7 @@ void AccountSettings::refreshSelectiveSyncStatus()
 
         ui->selectiveSyncButtons->setVisible(true);
         ui->bigFolderUi->setVisible(false);
+        ui->selectiveSyncApply->setEnabled(_model->isDirty());
```

Comment:
Since selectiveSyncApply is a child of selectiveSyncButtons it's unnecessary to setEnabled() on it in the bigFolder case.